examples/react-graph-gl/simple-graph-viewer/index.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+
+// graph.gl
```

Comment:
Please remove any string that exactly matches "graph.gl"

Reply:
Remove this one. Kept one in the README that mentions the history of the project. No others remain.